h11/tests/test_connection.py

```diff
@@ -59,7 +59,7 @@ def req(cl=None, te=False):
             assert _body_framing(meth, r) == ("content-length", (0,))
 
     # Transfer-encoding
-    for kwargs in [{"te": True}, {"cl": 100, "te": True}]:
```

Comment:
I removed this from the tests for `_body_framing` because it doesn't appear to be testing anything in particular and was conflicts with the new changes. `_body_framing` returns as soon as it hits 'Transfer-Encoding' so 'Content-Length' was never reached in this test anyway.